explorer/src/commitUtils.test.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import {userWeightForPath} from './commitUtils';
+
+const exampleData = {
+  fileToCommits: {
+    'foo.txt': ['1'],
+    'bar.txt': ['1', '2']
+  },
+  commits: {
+    '1': {
+      'author': 'dandelionmane',
+      'stats': {
+        'foo.txt': {'lines': 5, 'added': 3, 'deleted': 2},
+        'bar.txt': {'lines': 100, 'added': 100, 'deleted': 0}
+        }
+      },
+    '2': {
+      'author': 'wchargin',
+      'stats': {
+      'bar.txt': {'lines': 100, 'added': 50, 'deleted': 50},
+        }
```

Comment:
nit: could you reindent this object literal, please? (e.g., in vim: press `=ap`)